Realm/RLMUtil.mm

```diff
@@ -238,3 +237,18 @@ id RLMValidatedObjectForProperty(id obj, RLMProperty *prop, RLMSchema *schema) {
     }
     return outArray;
 };
+
+NSException *RLMException(NSString *reason, NSDictionary *userInfo) {
+    NSMutableDictionary *info = [NSMutableDictionary dictionaryWithDictionary:userInfo];
```

Comment:
That's what `dictionaryWithDictionary` is for, as opposed to `mutableCopy` -- it works when the parameter is `nil`.